### PR TITLE
Update Readme for event forwarder.

### PR DESCRIPTION
**Pull request by wildcardlinux**
No longer using Insert Keys and we also have an ARM template now.

https://docs.newrelic.com/docs/logs/forward-logs/azure-log-forwarding/#azure-eventhub

### CODE REVIEW

File: EventHubForwarder/README.md [hunk 1]
```diff
@@ -20,7 +20,7 @@ Parameters that can be configured in your Azure Resource Manager Template
 
 | Parameter  | Required | Default Value | Description
 |---|---|---|---|
-| New Relic License Key  | yes | `none` | Your New Relic Insights [insert key](https://docs.newrelic.com/docs/insights/insights-api/get-data/query-insights-event-data-api#register) |
+| New Relic License Key  | yes | `none` | Your New Relic [License_key](https://docs.newrelic.com/docs/logs/forward-logs/azure-log-forwarding/#azure-eventhub) can be created via ARM template. |
```
Comment (jsubirat) [inline EventHubForwarder/README.md:23]: Please point this link to https://docs.newrelic.com/docs/apis/intro-apis/new-relic-api-keys/ instead.

**Review by jsubirat — COMMENTED** [review]
Many thanks for your contribution @wildcardlinux ! Can you make the suggested changes? Thank you.

File: EventHubForwarder/README.md (hunk 1 above)
Comment (wildcardlinux) [reply EventHubForwarder/README.md]: Resubmitted.

**Review by jsubirat — APPROVED** [review]

**Comment by danybmx** [comment]
Thanks @wildcardlinux for the contribution!

**Pull request merged**